azuredevops/internal/service/feed/resource_feed.go

```diff
@@ -0,0 +1,247 @@
+package feed
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/feed"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/webapi"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/client"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils"
+)
+
+func ResourceFeed() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceFeedCreate,
+		Read:   resourceFeedRead,
+		Update: resourceFeedUpdate,
+		Delete: resourceFeedDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringIsNotWhiteSpace,
+				Required:     true,
+				ForceNew:     true,
+			},
+			"project_id": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.IsUUID,
+				Optional:     true,
+				ForceNew:     true,
+			},
+			"features": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"permanent_delete": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  true,
```

Comment:
it is recommended to set the default value to `false` to be consistent with the portal